demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -599,6 +617,96 @@ def _get_invalid_dbot_outputs(context_outputs_paths, context_outputs_description
 
         return missing_outputs, missing_descriptions
 
+    def validate_all_reputation_outputs_exist(
+        self, command: dict, output_results: OutputsResults
```

Comment:
instead of the object, just pass the two sets.

Reply:
done.